### PR TITLE
DAOS-9626 test: fix no_cap test timeout

### PR DESCRIPTION
**Pull request by wangshilong**
1. reduce pool size so that rebuild could be restarted
more quickly.

2. reduce cart timeout to make IV update return timeout
quickly to ease test.

Test-tag-hw-medium: no_cap
Test-repeat-hw-medium: 50
Signed-off-by: Wang Shilong <shilong.wang@intel.com>

### CODE REVIEW

**Comment by wangshilong** [comment]
@daltonbohning any idea how i could run specified medium hardware test several times? following metric did not work:
Test-tag-hw-medium: no_cap
Test-repeat-hw-medium: 50

File: src/tests/ftest/rebuild/no_cap.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ timeout: 180
 server_config:
   engines_per_host: 2
   name: daos_server
+  env_vars:
+    - CRT_TIMEOUT=10
```
Comment (saurabhtandan) [inline src/tests/ftest/rebuild/no_cap.yaml:14]: Can we please add some comments above this value change to help somebody looking at the test why this change is being made. It seems like , this is a workaround, so a slightly detailed description why we need to do this and when it should be removed will be helpful.
Also, it will be nice if a new ticket is opened (if not already created) to solve the real problem as this is a workaround.

**Comment by wangshilong** [comment]
@liw @wangdi1 it looks it could pass test, would you help review? thanks!

**Review by liw — APPROVED** [review]
I was not on the list of reviewers.

**Review by wangdi1 — APPROVED** [review]

**Review by daltonbohning — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM
